packages/manager/modules/vps/src/dashboard/vps-dashboard.controller.js

```diff
@@ -57,6 +57,14 @@ export default class {
   }
 
   $onInit() {
+    this.expirationDate = moment(this.serviceInfo?.expiration).format(
+      'DD/MM/YYYY',
```

Comment:
Shouldn't we use something like `L` or `LL` as in some languages the display is `MM/DD/YYYY`